apps/site/assets/ts/stop/components/amenities/AmenityCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderFa from "../../../helpers/render-fa";
+
+const AmenityCard = ({
+  headerText,
+  icon
+}: {
+  headerText: string;
+  icon: JSX.Element;
+}): JSX.Element => {
+  return (
+    <div className="d-flex card justify-content-space-between">
+      <div className="m-16">
+        <div className="d-flex text-primary">
+          <div className="">{icon}</div>
```

Comment:
```suggestion
          {icon}
```
I believe since it's in a `display: flex` the icon will render properly even without the wrapper `<div>`.